debug/index.js

```diff
@@ -0,0 +1,95 @@
+const border = 25;
```

Comment:
Remove this and move a cleaned up version to a dedicated page?

Reply:
done in 69cf025